XMonad/Prompt.hs

```diff
@@ -228,6 +228,15 @@ class XPrompt t where
 
 data XPPosition = Top
                 | Bottom
+                -- | Prompt will be placed in the center horizontally and
+                --   in the certain place of screen vertically. If it's in the upper
+                --   part of the screen, completion window will be placed below(like
+                --   in 'Top') and otherwise above(like in 'Bottom')
+                | CenteredAt { xpCenterY :: Rational
+                             -- ^ Relative Y coordinate of center of the prompt.
```

Comment:
My concern was more that the units are not specified.  I suppose saying
"relative" plus the fact that it is a Rational value should be a hint that
it should be a value from 0 to 1 (as opposed to, say, a number of pixels).
But it would still be nice to say this explicitly.  "A rational between 0
and 1, giving the relative..."

On Fri, Jan 8, 2016 at 11:26 AM Bogdan Sinitsyn <notifications@github.com>
wrote:

> In XMonad/Prompt.hs
> <https://github.com/xmonad/xmonad-contrib/pull/29#discussion_r49213596>:
>
> > @@ -228,6 +228,15 @@ class XPrompt t where
> >
> >  data XPPosition = Top
> >                  | Bottom
> > +                -- | Prompt will be placed in the center horizontally and
> > +                --   in the certain place of screen vertically. If it's in the upper
> > +                --   part of the screen, completion window will be placed below(like
> > +                --   in 'Top') and otherwise above(like in 'Bottom')
> > +                | CenteredAt { xpCenterY :: Rational
> > +                             -- ^ Relative Y coordinate of center of the prompt.
>
> "relative" means relative to screen width for prompt width and relative to
> screen hight for y coordinate. i did't think that this could be understood
> in other way
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xmonad/xmonad-contrib/pull/29/files#r49213596>.
>